students/scotchwsplenda/Lesson_5/MailRoom_3.py

```diff
@@ -26,80 +25,87 @@ def main():
                         3: exit_program, 4: mass_mail}
             swit_dic[choice]()
             break
+        except KeyError:
```

Comment:
you don't want to have a `break` above -- you want your program to continue prompt user until they choose to exit.

the rest of main() function calls need to be removed.